### PR TITLE
dotCMS/core#23591 Container layout shift when load

### PR DESCRIPTION
**Pull request by zulqarnainvd**
### Proposed Changes
* fix layout issue when container load

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **
[screen-capture (10).webm](https://user-images.githubusercontent.com/113915849/212896828-8a7384f4-1311-4f05-95cc-78d1d2ab41d0.webm)|[screen-capture (9).webm](https://user-images.githubusercontent.com/113915849/212896497-1223bd86-b8c7-41af-bfca-48754102b2c8.webm)

### CODE REVIEW

**Comment by fmontes** [comment]
https://user-images.githubusercontent.com/751424/212991200-445d7ee0-5309-4e3b-aa36-8483b31cc38f.mp4

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/store/dot-container-properties.store.ts [hunk 1]
```diff
@@ -63,9 +63,9 @@ export class DotContainerPropertiesStore extends ComponentStore<DotContainerProp
             )
             .subscribe((containerEntity: DotContainerEntity) => {
                 const { container, contentTypes } = containerEntity;
-                if (container && (container.preLoop || container.postLoop)) {
+                if (container && contentTypes?.length > 0) {
                     this.updatePrePostLoopAndContentTypeVisibility({
-                        showPrePostLoopInput: true,
+                        showPrePostLoopInput: !!container.preLoop || !!container.postLoop,
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/store/dot-container-properties.store.ts:68]: See the video in the conversation tab.

**Review by fmontes — CHANGES_REQUESTED** [review]

**Comment by zulqarnainvd** [comment]
[screen-capture (4).webm](https://user-images.githubusercontent.com/113915849/213696006-f0e436f9-1b17-42a4-93cf-fe5678f7b012.webm)

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.spec.ts [hunk 2]
```diff
@@ -323,5 +327,13 @@ describe('DotContentEditorComponent', () => {
                 expect(comp.monacoEditors[mockContentTypes[0].id].focus).toHaveBeenCalled();
             }));
         });
+
+        fit('shoud have add loader on content types', () => {
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.spec.ts:331]: Remove fit.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.ts [hunk 3]
```diff
@@ -45,6 +51,20 @@ export class DotContentEditorComponent implements OnInit {
...
+        });
+
+        this.init();
+        this.updateActiveTabIndex(this.getcontainerStructures.length);
+        if (changes.contentTypes.currentValue.length > 0) {
+            Object.keys(this.monacoEditors).forEach((editorId) => {
+                this.monacoEditors[editorId].updateOptions({ readOnly: false });
+            });
+        }
+    }
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.ts:66]: How many times this gets called in a lifecycle?

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.ts (hunk 3 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.ts:66]: maximum 2 time only

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.spec.ts (hunk 2 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.spec.ts]: updated

**Review by fmontes — APPROVED** [review]

**Pull request merged**